### ISSUE TITLE
show whole variable declaration

### ISSUE THREAD

**Issue by Morgy93** [felixfbecker/php-language-server#446]
Hello,

is it possible to show the whole variable declaration within the tooltip instead of just the first line?

Example:
![image](https://user-images.githubusercontent.com/7961978/28662872-e6d45f28-72bb-11e7-8b95-2364023ba386.png)

**Comment by felixfbecker** [comment]
Depends on https://github.com/Microsoft/tolerant-php-parser/issues/138

cc @roblourens

**Comment by roblourens** [comment]
What's the expected result? I don't think we should show everything passed to `array`. TS shows the variable name with type annotation.

**Comment by felixfbecker** [comment]
@roblourens Here's a nice example from Go:

![image](https://user-images.githubusercontent.com/10532611/29051631-f1e09702-7b98-11e7-9bdd-75a21e4aa871.png)

I remember that being able to see the struct contents was a very highly requested feature in Go

**Comment by roblourens** [comment]
Is that from a sourcegraph extension to the LSP?

**Comment by felixfbecker** [comment]
No, it just uses 3 hover contents: https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#textDocument_hover
